src/app/views/query-response/snippets/snippets-helper.tsx

```diff
@@ -123,6 +128,36 @@ function Snippet(props: ISnippetProps) {
     );
   }
 
+  const displayError = (): JSX.Element | null => {
+    if((!loadingState && snippet) || (!loadingState && !snippetError)){
+      return null;
+    }
+    // OpenAPI generated and C# snippets are returning 404 for snippets not available
+    // other snippets are returning 400 for snippets not available
+    // https://github.com/microsoftgraph/microsoft-graph-devx-api/issues/1494
+    if(
+      (openApiSnippets.includes(language) && snippetError?.status && snippetError.status === 404) ||
+      (snippetError?.status && snippetError.status === 400)
+    ){
+      return(
+        <Label style={{ padding: 10 }}>
+          <FormattedMessage id='Snippet not available' />
+        </Label>
+      )
+    }
+    else{
+      return (
+        <>
+          {!loadingState &&
+            <Label style={{ padding: 10 }}>
+              <FormattedMessage id='Fetching code snippet failing' />
```

Comment:
This part tells the user's to retry... is this what we want?

Reply:
This uncovered a very hidden edge case that I missed. Thank you @ElinorW